tests/unittests/test_commands.py

```diff
@@ -134,6 +134,28 @@ def test_text_saves_params(self):
         self.assertEqual(output_text.font, "Bitstream Vera Bold Italic")
         self.assertEqual(output_text.fontsize, 64)
 
+    @test_as_bot()
+    def test_text_bounds_property(self):
+        """
+        Check text.bounds() against expected values
+        """
+        test_fonts = {
+            "DejaVu Sans Book": (1, 87.0, 87, 14),
+            "Liberation Sans Regular": (1, 88.0, 79, 13),
+            "Bitstream Vera Sans Roman": (1, 87.0, 87, 14),
+        }
```

Comment:
I'd pass the font in via @parameterize - see the other tests in this file, it should simplify this.

Reply:
done -- not sure how to do the available font check without making a duplicate list, though